app/scripts/services/dimCsvService.factory.js

```diff
@@ -0,0 +1,201 @@
+(function() {
+  'use strict';
+
+  angular.module('dimApp')
+    .factory('dimCsvService', CsvService);
+
+  CsvService.$inject = ['$rootScope', '$q', 'dimBungieService', 'dimItemDefinitions', 'dimStoreService', '$http'];
+
+  function CsvService($rootScope, $q, dimBungieService, dimItemDefinitions, dimStoreService, $http) {
+
+    function downloadCsv(filename, csv){
```

Comment:
Would you mind doing a pass through this file normalizing the code style to match the rest of DIM? Mostly just spacing (space before {, space around operators, etc), no brace-less if statements, etc.